example/src/App.tsx

```diff
@@ -1,19 +1,30 @@
-import * as React from 'react';
+import React, { useState } from 'react';
 
 import { StyleSheet, View, Text } from 'react-native';
-import { configure } from '@chargebee/react-native-chargebee';
+import Chargebee from '@chargebee/react-native-chargebee';
 
 export default function App() {
-  const [result, setResult] = React.useState<number | undefined>();
+  const [configComplete, isConfigComplete] = useState(false);
+  const site = 'site';
+  const apiKey = 'test_key';
+  const sdkKey = 'sdk-key';
 
   React.useEffect(() => {
-    console.log("Configuring Site with: Site, apiKey, sdkKey")
-    configure("Site", "apiKey", "sdkKey")
+    Chargebee.configure({
```

Comment:
Maybe for later once this gets more shape, but I guess we'll have a try/catch here to handle failed config?

Reply:
Will add this one along side.